colin/checks/best_practices/no_root.py

```diff
@@ -0,0 +1,25 @@
+from colin.checks.abstract.containers import ContainerCheck
+from colin.checks.abstract.images import ImageCheck
+from colin.checks.result import CheckResult
+
+
+class NoRootCheck(ContainerCheck, ImageCheck):
+
+    def __init__(self):
+        super().__init__(name="no_root",
+                         message="Service should not run as root by default.",
+                         description="",
+                         reference_url="?????",
+                         tags=["root", "user"])
+
+    def check(self, target):
+        metadata = target.instance.get_metadata()["Config"]
+        root_present = "User" in metadata and metadata["User"] and metadata["User"] != "root"
```

Comment:
I'm thinking whether there could be `"0"` too.

Reply:
Good question. Probably yes.

Reply:
I've added the `0` too.